### ISSUE TITLE
App name: "Thorium Reader" with space problematic in CLI for Mac, Windows, etc. to pass command line parameters

### ISSUE THREAD

**Issue by danielweck** [readium/readium-desktop#411]
Related PR: https://github.com/readium/readium-desktop/pull/403

**Comment by llemeurfr** [comment]
No problem to use "thorium" for CLI usage. Adding "Reader" is useful for discovery on stores and on the web.

**Comment by danielweck** [comment]
Suggestion:
* For MacOS: rename  `productName` `Thorium Reader` to `Thorium` https://github.com/readium/readium-desktop/blob/f0a751c5fc5e1640bfa99259cd8e6a94591ec7ad/package.json#L63
* For Linux: rename `executableName` `Thorium Reader` to `Thorium` https://github.com/readium/readium-desktop/blob/f0a751c5fc5e1640bfa99259cd8e6a94591ec7ad/package.json#L121
* For Windows: preserve `displayName` `Thorium Reader` BUT verify that the new `productName` does not mess-up our existing Windows App Store deployment!! (risk of regression bug due to altered app identity?) https://github.com/readium/readium-desktop/blob/f0a751c5fc5e1640bfa99259cd8e6a94591ec7ad/package.json#L128

**Comment by llemeurfr** [comment]
Suggestion from Daniel accepted

**Comment by danielweck** [comment]
I have revised my assessment, based on recent CLI tests on Windows, Linux and Mac, and also based on what we envision the real-world usage to be (i.e. expert users, advanced functionality available from the command line shell).
I think "Thorium Reader" is fine in most cases, except for Linux where it is more common to invoke an app from the console (similar to how VisualStudioCode can be launched from the shell using `code`). So, on Linux the standalone portable AppImage will work just fine, but the installed Debian package will work better with `executableName` = `thorium-reader` or `thorium` (the former is probably a better choice to avoid possible conflicts with the common `thorium` term potentially found in the environment executable path).
For Windows I am concerned that changing the `productName` will alter the application identity and cause problems in the App Store. For MacOS I am not concerned about CLI usage with the current "Thorium Reader" name.
So to summarize: the only change I would recommend is ` "executableName": "thorium-reader"`.

**Comment by danielweck** [comment]
Related issue: https://github.com/readium/readium-desktop/issues/547

**Comment by danielweck** [comment]
Related PR: https://github.com/readium/readium-desktop/pull/651
